### PR TITLE
Fix the link

### PR DESCRIPTION
**Pull request by Antall**

### CODE REVIEW

File: _includes/docs/2.5/database-setup.html [hunk 1]
```diff
@@ -43,7 +43,7 @@ <h2 id="database-setup">Database Setup</h2>
 </pre>
 
 <h3>3. Creating the Azkaban Tables</h3>
-<p>Download the azkaban-sql-script tarball from the <a href="###">download page</a>. Contained in this archive are table creation scripts.</p>
+<p>Download the azkaban-sql-script tarball from the <a href='../../downloads.html'>download page</a>. Contained in this archive are table creation scripts.</p>
```
Comment (kaneda) [inline _includes/docs/2.5/database-setup.html]: Relative links for a web page make me cringe.

Comment (Antall) [inline _includes/docs/2.5/database-setup.html]: That change is in keeping with the other links defined. That said I'll make the necessary change.

**Comment by erwa** [comment]
I followed the guide here: https://help.github.com/articles/using-jekyll-with-pages/

Something like:
```
gem install bundler
```

Create a `Gemfile`
```
source 'https://rubygems.org'
gem 'github-pages'
```

Run:
```
bundle install
bundle exec jekyll serve
```

Then view the page at http://localhost:4000.

**Comment by Antall** [comment]
Validated and it works. Thanks for the help.

**Comment by erwa** [comment]
Thanks, Antall! Merging.

**Pull request merged**